maploom/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = (1, 5, 60)
```

Comment:
Unsure how this project is built to update version number - will Jenkins need to be modified for this or something else?